scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -938,6 +949,41 @@ def apply_filters(self, **kwargs):
                 # handle quarantined tests
                 self.handle_quarantined_tests(instance, plat)
 
+                # Check if scripting is available
```

Comment:
shouldn't this code, which is not a filter, be called from `load()` to work with `--load_tests` and `--test_only` use cases as well ?

Reply:
done